logging_config.py

```
@@ -0,0 +1,497 @@
+        r'sk-[a-zA-Z0-9]{48}',
+        r'pk-[a-zA-Z0-9]{48}',
+        r'[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}',  # Email addresses
+        r'\b\d{3}-\d{2}-\d{4}\b',  # SSN pattern
+        r'\b\d{4}[\s-]?\d{4}[\s-]?\d{4}[\s-]?\d{4}\b',  # Credit card pattern
+    ]
+    
+    def __init__(self):
+        super().__init__()
+        self.patterns = [re.compile(pattern, re.IGNORECASE) for pattern in self.SENSITIVE_PATTERNS]
+    
+    def filter(self, record):
```

Ruff (0.12.2)

Missing return type annotation for public function `filter`

Add return type annotation: `bool`

(ANN201)